mlflow/genai/judges/utils.py

```diff
@@ -33,6 +33,30 @@ def format_prompt(prompt: str, **values) -> str:
     return prompt
 
 
+def add_output_format_instructions(prompt: str) -> str:
+    """
+    Add structured output format instructions to a judge prompt.
+
+    This ensures the LLM returns a JSON response with 'result' and 'rationale' fields,
+    matching the expected format for the invoke_judge_model function.
+
+    Args:
+        prompt: The formatted prompt with template variables filled in
+
+    Returns:
+        The prompt with output format instructions appended
+    """
+    output_format_instructions = """
+
+Please provide your assessment in the following JSON format only (no markdown):
+
+{
+    "result": "Your evaluation result/rating",
+    "rationale": "Detailed explanation for your evaluation"
+}"""
```

Comment:
@BenWilson2 We should derive this from Judge.get_output_fields(), rather than hardcoding. See https://github.com/mlflow/mlflow/pull/17502/files#diff-2cd05ad709d0d3fdf0757afab20f023ee8b12ad82308ed898b1d8c4368e32fe3R186-R189